src/Maker/MakeForm.php

```diff
@@ -95,7 +95,14 @@ public function generate(InputInterface $input, ConsoleStyle $io, Generator $gen
                 'Entity\\'
             );
 
-            $doctrineEntityDetails = $this->entityHelper->createDoctrineDetails($boundClassDetails->getFullName());
+            try {
+                $doctrineEntityDetails = $this->entityHelper->createDoctrineDetails($boundClassDetails->getFullName());
+            } catch (\Exception $e) {
+                // DoctrineBundle isn't installed
+                $io->comment($e->getMessage());
+
+                $doctrineEntityDetails = null;
+            }
```

Comment:
Not sure that keeping the io output (_Somehow the doctrine service is missing. Is DoctrineBundle installed?_) is usefull. 
You probably know that doctrine is not installed and if not, this may not be the most appropriate moment to indicate it.

```suggestion
            } catch (\Exception $e) {
                $doctrineEntityDetails = null;
            }
```